politeiawww/backend.go

```diff
@@ -1476,11 +1476,12 @@ func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, isUse
 		return &reply, nil
 	}
 
-	// The title and files for unvetted proposals should not be viewable by
+	// The files for unvetted proposals should not be viewable by
 	// non-admins; only the proposal meta data (status, censorship data, etc)
 	// should be publicly viewable.
 	if !isVettedProposal && !isUserAdmin {
 		reply.Proposal = www.ProposalRecord{
+			Name:             cachedProposal.Name,
```

Comment:
The name actually shouldn't be viewable to everyone, just to the user who submitted the proposal (and to admins).